### PR TITLE
Friday's changes

### PR DESCRIPTION
**Pull request by Agubuzoboy**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -3,10 +3,15 @@ Description:
 The analyzers can potentially support different languages and different policies, enabling this tool to provide wide and deep scans across a repository with very little effort from the end user 
 
 Installation: 
-    The user first needs to download this repository and install it somewhere in their home directory. 
+    The user first needs to download this repository. 
     Python3, pip, and java are required dependecies 
-    At this point the user should be ready to go 
-
+    At this point the user should be ready to go  
+    The tool automatically installs the static analyzer dependecies (th estatic analyzers used in this tool). Here is a list of the current installations
```
Comment (devadigar) [inline readme.txt]: Fix Typo.

File: readme.txt [hunk 2]
```diff
@@ -33,8 +38,8 @@ Usage:
         3. a verbose json report which delineates all of the issues found by the scans. This can be found in the reports directory 
     After every session of scans a new set of these three reports are generated. The latter two being files differentiated from other scans by timestamps. 
 
-
-
+WARNINGS    
+    Currently it is not recommended to run this program parallely due to certain static analyzers interacting withthem selves
```
Comment (devadigar) [inline readme.txt:47]: I remember we discussed about adding additional details around debugging commands for checking sonarqube server status, on which port sever is running, how to bring up server manually, if needed etc...

You can separate heading about Sonarqube and these details

File: readme.txt [hunk 3]
```diff
@@ -3,10 +3,15 @@ Description:
 The analyzers can potentially support different languages and different policies, enabling this tool to provide wide and deep scans across a repository with very little effort from the end user 
 
 Installation: 
-    The user first needs to download this repository and install it somewhere in their home directory. 
+    The user first needs to download this repository. 
     Python3, pip, and java are required dependecies
```
Comment (devadigar) [inline readme.txt]: To make it clear, you can add these pre-req under separate "Pre-requisites" section

**Review by devadigar — CHANGES_REQUESTED** [review]
2 small ReadME changes needed, otherwise looks good.

**Review by devadigar — APPROVED** [review]

**Review by devadigar — APPROVED** [review]

**Review by xuchenma — APPROVED** [review]

**Pull request merged**